### DISCUSSION TITLE
contact forum

### DISCUSSION THREAD

**Discussion by iamparas0** [iamparas0/TIC-TAC-TOE#141] [General]
everyone can ask me related to project here.

**Comment by Techbarsha** [comment]
add more features like Difficulty Levels,win and loss sound,Score Tracking and Stats and Hints and Assistance..
can you pls assign this to me!!I believe it will improve the visual appeal of the project.

**Reply by iamparas0** [reply]
i appreciate it , i can only assign you a specific topic cause there lots of contributor are there who also wants to contribute  so i feel they also can contribute . if i assign you all that won't be fare with them right.

please raise issue related to Difficulty Levels , Hints and Assistance. i will assign you for this
